media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="It provides a toolkit for video editors and many more applications"
```

Comment:
Could you update the description to match the one from the readme.md?
```
MLT is a LGPL multimedia framework designed for television broadcasting, and melted is a GPL multi-unit video layout server with realtime effects.
```
Mind the 80 char limit